rosa_support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="showing-data-collected-by-remote-health-monitoring"]
+= Showing data collected by remote health monitoring
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: showing-data-collected-by-remote-health-monitoring
+
+toc::[]
+
+As an administrator, you can review the metrics collected by Telemetry and the Insights Operator.
+
+include::modules/telemetry-showing-data-collected-from-the-cluster.adoc[leveloffset=+1]
+
```

Comment:
The ROSA part in [ROSA Showing data collected by remote health monitoring](https://55445--docspreview.netlify.app/openshift-rosa/latest/rosa_support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.html)  is not readable
You have installed the Red Hat OpenShift Service on AWS CLI (oc)

Reply:
Thank you for catching that! I added conditionals so it should always render as `OpenShift Container Platform` since it's for the `oc` CLI.